src/output/file.rs

```diff
@@ -28,12 +28,13 @@ use maplit::{btreemap, convert_args};
 ///
 /// ```
 /// # use ocptv::output::*;
+/// # use std::str::FromStr;
 ///
 /// let uri = Uri::parse("file:///tmp/foo").unwrap();
 /// let file = File::builder("name", uri)
 ///     .is_snapshot(true)
 ///     .description("description")
-///     .content_type("text/plain")
+///     .content_type(Mime::from_str("text/plain").unwrap())
```

Comment:
https://docs.rs/mime/latest/mime/constant.TEXT_PLAIN.html

the point was to not have arbitrary strings

Reply:
need to fix this before merge